packages/react-params/src/index.tsx

```diff
@@ -85,10 +85,7 @@ class Params extends React.PureComponent<Props, State> {
     if (values && params) {
       params.map(({ type }: ParamDef, index: number) => {
         if (type.type?.includes('AssetId')) {
-          this.setState(prevState => ({
-            ...prevState,
-            assetIdContext: values[index].value.toString(),
-          }))
+          this.state.assetIdContext = values[index].value.toString();
```

Comment:
using `setState` ended up breaking this

Reply:
https://reactjs.org/docs/react-component.html#setstate
React docs says - Never mutate this.state directly, as calling setState() afterwards may replace the mutation you made. Treat this.state as if it were immutable.

Reply:
not sure why I had this in render function 🤦‍♂️